README.md

```diff
@@ -173,9 +173,6 @@ Run `./dss-api` in the top-level `data-store` directory to deploy the DSS API on
 
 ### Deploying the DSS
 
-Assuming the tests have passed above, the next step is to manually deploy. See the section below for information on
-CI/CD with Travis if continuous deployment is your goal.
-
```

Comment:
Do you think the Travis section should be removed?

Reply:
Oh, sorry about that, it was just the **ordering** of the sections that was off... 1) deploy, 2) passing tests, not the other way around.

Reply:
The intention is that some tests should pass locally before deploying. Generally we have two types of tests:

- "standalone tests" which should not depend on deployed components
- "integration tests" which depend upon deployed components

It's reasonable to expect standalone tests to pass before attempting deployment.